mlir/lib/Dialect/MemRef/Transforms/NormalizeMemRefs.cpp

```diff
@@ -347,6 +358,11 @@ void NormalizeMemRefs::normalizeFuncOpMemRefs(func::FuncOp funcOp,
   for (memref::AllocOp allocOp : allocOps)
     (void)normalizeMemRef(&allocOp);
 
+  SmallVector<memref::AllocaOp, 4> allocaOps;
```

Comment:
```suggestion
  SmallVector<memref::AllocaOp> allocaOps;
```

Perhaps also do the same above ? :)